### ISSUE TITLE
Extra Copy of jQuery Inside Project

### ISSUE THREAD

**Issue by TCB13** [DataTables/DataTables#586]
Hello,

I just installed DataTables for the first time using Bower. It all went well, however I noticed there's a `jquery.js` at `/datatables/media/js` why is this file there?

Since DataTables declares jQuery as a dependency at `bower.json`:
````````
"dependencies": {
        "jquery": ">=1.7.0"
},
````````

Why do we need a copy of jQuery inside the project itself? Can't that `jquery.js` be removed from the DataTables project, or, at least the `bower.json` file modified to ignore it?

Thank you.

**Comment by DataTables** [comment]
It's there for the convenience of those who aren't using bower or other package managers. Ultimately it probably can be removed now.

Bower doesn't automatically include it in your HTML does it? It is just an extra unused file so isn't actually causing any issues or conflicts?

**Comment by TCB13** [comment]
Bower doesn't automatically include anything.
This is just an extra unused file, wasted space basically.

**Comment by DataTables** [comment]
Fair enough - I'll look into changing as part of the updates I need to make to this repo!

Thanks for flagging this up.

**Comment by TCB13** [comment]
You're welcome.
